docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -1429,6 +1429,40 @@ How it **works**:
 
 - The series must already exist before exemplars can be appended, as we do not create new series upon ingesting exemplars. The series will be created when a sample from it is ingested.
 
+### err-mimir-store-consistency-check-failed
+
+This error occurs when the querier is unable to fetch some of the expected blocks after multiple retries and connections to different store-gateways. The query fails because some blocks are missing.
```

Comment:
I would clarify they're not missing in the storage, but just in the store-gateways.

```suggestion
This error occurs when the querier is unable to fetch some of the expected blocks after multiple retries and connections to different store-gateways. The query fails because some blocks are missing in the queried store-gateways.
```